src/wayback/_utils.py

```diff
@@ -322,26 +320,25 @@ def __exit_all__(self, type, value, traceback):
         pass
 
 
-class DisableAfterCloseSession(requests.Session):
+class DisableAfterCloseAdapter:
```

Comment:
I wound up keeping this but changing the implementation slightly (so it mixes in kind of like `RetryAndRateLimitAdapter` does). It should probably move to `_http.py`.